README.md

```diff
@@ -9,7 +9,7 @@ A very simple way of sticking a read only REST API on a single mongoDB database.
 
 ### How do I use it?
 
-It's just a bottle app, so you can deploy it the same way you would any bottle app.  You'll also need to set the mongoDB connection URI and the name of the target database in the configuration.  An example configuration is included in the repository.
+It's just a bottle app, so you can deploy it the same way you would any bottle app.  You'll also need to set the mongoDB connection URI and the name of the target database in the configuration.  An example configuration is included in the repository. It is also possible to use environment variables to configure Kala, the key variables are `KALA_MONGODB_URI`, `KALA_MONGODB_DB` and `KALA_CONFIGFILE`.
```

Comment:
Please document which "wins" in the case of a conflict between environment variables and configuration file.